### PR TITLE
Show Avatar image for Workspace List Item in Sidebar

### PR DESCRIPTION
**Pull request by mananjadhav**
@nickmurray47 Can you take a look this PR?

### Details
Updated the `MenuItem` component to allow Avatar support and accordingly updated the Workspace List Item icon param.

### Fixed Issues

$ #4434 

### Tests
1. Added Avatar Image and checked the sidebar list item
2. Tested removing Avatar image should rollback to show a default icon
3. Default Icon to be the building icon

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web
https://user-images.githubusercontent.com/3069065/128763484-28ad0b1b-2afe-4c10-b5f0-fe578d1a6d5b.mp4

#### Mobile Web
wkspace-sbar-icon-mweb

#### Desktop
wkspace-sbar-icon-desktop

#### iOS
wkspace-sbar-icon-ios

#### Android
wkspace-sbar-icon-android

### CODE REVIEW

File: src/components/MenuItem.js [hunk 1]
```diff
@@ -59,6 +60,8 @@ const propTypes = {
 
     /** A right-aligned subtitle for this menu option */
     subtitle: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+
+    iconType: PropTypes.oneOf(['avatar', 'icon']),
```
Comment (nickmurray47) [inline src/components/MenuItem.js:64]: Missing comment here

**Comment by nickmurray47** [comment]
Looks pretty good to me, just need to add one comment there and this looks ready to go!

File: src/components/MenuItem.js (hunk 1 above)
Comment (mananjadhav) [reply src/components/MenuItem.js:65]: Added

**Comment by mananjadhav** [comment]
PR Updated

**Review by MonilBhavsar — DISMISSED** [review]

**Comment by mananjadhav** [comment]
@nickmurray47 Can you check the PR again and merge this? Monil has approved the changes.

**Review by nickmurray47 — CHANGES_REQUESTED** [review]
Last thing @mananjadhav but can you replace any references to the strings 'avatar' or 'icon' with new variables added to the `CONST.js` file please, i.e. `'avatar'` becomes `CONST.AVATAR`?

**Comment by mananjadhav** [comment]
@nickmurray47 Done. Thanks for pointing it out. I should've done it in the first place.

File: src/components/MenuItem.js [hunk 2]
```diff
@@ -113,7 +120,7 @@ const MenuItem = ({
         {({hovered, pressed}) => (
             <>
                 <View style={styles.flexRow}>
-                    {icon && (
+                    {(icon && iconType === 'icon') && (
```
Comment (nickmurray47) [inline src/components/MenuItem.js:123]: here and below
```suggestion
                    {(icon && iconType === 'CONST.ICON_TYPE_AVATAR') && (
```

Comment (mananjadhav) [reply src/components/MenuItem.js]: Good eye.  Fixed.

**Review by nickmurray47 — APPROVED** [review]
👍

**Comment by botify** [comment]
This has been deployed to production and is now subject to a 7-day regression period.
If no regressions arise, payment will be issued on 2021-09-06.  :confetti_ball:

**Pull request merged**